src/modules/handlers/info.ts

```diff
@@ -0,0 +1,25 @@
+// eslint-disable-next-line no-unused-vars
+import Koa from 'koa'
+import * as mongoose from 'mongoose'
+import { UserSchema } from '../../models/user'
+
+const User = mongoose.model('User', UserSchema)
+
+async function createUsers () {
+  await User.create({
+    name: 'Carolina Estrella',
+    email: 'carolina@host.com.br'
+  })
+}
+
+async function getUsers () {
+  return User.find()
+}
+
+createUsers()
```

Comment:
Yes, and that is a bad programming pratice

Reply:
@carolestrella, do you think using this can cause problems ? Explain why

Reply:
Yes, it was supposed to be created inside a function and not in file. I changed it to `async function info (ctx: Koa.Context) : Promise<void>`

Reply:
Why it supposed to be created inside and why it can cause problems ?

Reply:
It is supposed to be created when I want to call `createUsers()` function. The way it was before, it was running `createUsers()` immediately when i ran Yarn dev, which was a bad choice because I hadn't had control over it.